vignettes/advanced_data.Rmd

```diff
@@ -0,0 +1,211 @@
+library(ggplot2)
+data(pneumonic_plague)
+summary(pneumonic_plague)
+```
+
+## Conditioning on observable chain sizes
+
+Among the data, we see a number of relatively small chains, but no singletons.
+This is because when Nishiura _et al._ investigated outbreaks of the pneumonic plague they compiled data only on chains of at least 2 cases.
+
+In general, any particular dataset may include chains observed only of some sizes.
+Analyzing such data requires appropriately conditioning the likelihood.
+This is accomplished using the `condition_geq` argument, which specifies the minimum observation size _for each chain_ (allowing for a mix, if needed).
+For analyzing the pneumonic plague data, we need to set `condition_geq = rep(2, 19)`, indicating that all chains were counted only if they had at least 2 cases.
+One more wrinkle remains before analysis.
+
+## What if the chain didn't go extinct?
+
+We might also notice in that data that there is one very large chain of size 5009.
+This is not a stuttering chain.
+Nishiura _et al._ call it a major epidemic.
+We have two choices for how to appropriately handle this.
+Both amount to acknowledging that this chain did not go extinct due to any feature of the branching process model that this package uses, but differ in how they treat it.
+
+### Treat it as non-extinct
+
+We can simply declare that this chain did not go extinct.
+Mathematically this requires partitioning outcomes into exctinct and non-extinct chains and considering them separately.
+But in practice with `chainsR` this is done by declaring the size of a chain to be `Inf` (which is the package's shorthand for non-extinct).
+Note that this will force $R > 1$ because when $R < 1$ _every_ chain _must_ go extinct.
+
+```{r treat as inf, results=FALSE, warning=FALSE}
+pneumonic_plague_inf <- pneumonic_plague
+pneumonic_plague_inf[4] <- Inf
+
+fit_inf <- fit_nbbp_homogenous_bayes(
+  all_outbreaks = pneumonic_plague_inf,
+  condition_geq = rep(2, length(pneumonic_plague)),
+  seed = 42
+)
+```
+
+As always, first we inspect the MCMC.
+
+```{r plague inf mcmc}
+rstan::check_hmc_diagnostics(fit_inf)
+
+rstan::summary(fit_inf)$summary[, c("n_eff", "Rhat")]
+```
+
+We see a small number of divergent transitions, but things look good on the whole.
+
+Now we can look at what we estimated.
+
+```{r plague inf res}
+fit_inf
+```
+
+We can see that $R$ is inferred to be above one with probability 1.
+It is likely near 1, though.
+We see considerable uncertainty about $k$, as we did in the borealpox example.
+
+### Treat it as censored
+
+We can alternately declare that all we know is that chain was _at least_ as big as we saw.
+That is, it didn't go extinct under its own power at size 5009, but it might have at 5010, or 7500, or something else.
+This is accomplished using the `ccensor_geq` argument, which specifies the smallest possible size that _each chain_ (allowing for as many censored observations as needed), with `NA` meaning no censoring is applied.
```

Comment:
The clause "which specifies the smallest possible size that each chain" appears to be incomplete. Should it be "the smallest possible size _of_ each chain?"